[18.0][MIG] account_journal_lock_date: Migration to 18.0

StefanRijnhart opened the pull request:
Standard migration.
* Entry point was renamed in https://github.com/odoo/odoo/commit/c73d430080da602aa1528842b591040d0717987f#diff-1e3bd6be3bfb83a37ec9fb800ce8b1c95afe0be90ff792874ae7299c320a2f6e
* Test account_journal_restrict_mode separately as it is too restrictive.

sbidoul committed:
[ADD] account_journal_lock_date

sbidoul committed:
[FIX] account_journal_lock_date test on travis

The test failed on travis (not on a local install) because the
admin users if part of the Adviser group. Make sure we have the right
group and add a test for that feature too.

oca-transbot committed:
OCA Transbot updated translations from Transifex

oca-travis committed:
[UPD] Update account_journal_lock_date.pot

sbejaoui committed:
[MIG] - account_journal_lock_date migration to 12.0

Commit:
[IMP] account_journal_lock_date: two dates per journal

- Change in journal the existing 'Lock date' by two dates, the same as in company.
- Add a wizard to allows a massive update of those dates for several journals at the same time.

oca-travis committed:
[UPD] Update account_journal_lock_date.pot

Commit:
[IMP] account_journal_lock_date: black, isort, prettier

Commit:
[MIG] account_journal_lock_date: Migration to 13.0

oca-travis committed:
[UPD] Update account_journal_lock_date.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-13.0/account-financial-tools-13.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-13-0/account-financial-tools-13-0-account_journal_lock_date/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-13.0/account-financial-tools-13.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-13-0/account-financial-tools-13-0-account_journal_lock_date/

Zar21 committed:
[MIG] account_journal_lock_date: Migration to 14.0

oca-travis committed:
[UPD] Update account_journal_lock_date.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[IMP] update dotfiles

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 20.0% (4 of 20 strings)

Translation: account-financial-tools-14.0/account-financial-tools-14.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-14-0/account-financial-tools-14-0-account_journal_lock_date/it/

victoralmau committed:
[FIX] account_journal_lock_date: Add post_install tag in tests to prevent test errors since https://github.com/odoo/odoo/commit/81aac30dd2278e43a43cf8fd6cef31e1e8c60f3f
TT33774

OCA-git-bot committed:
account_journal_lock_date 14.0.1.0.1

Reyes4711-S73 committed:
[MIG] account_journal_lock_date: Migration to 15.0

Commit:
[UPD] Update account_journal_lock_date.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
account_journal_lock_date 15.0.1.0.1

RodrigoBM committed:
[MIG] account_journal_lock_date: Migration to 16.0

alexis-via committed:
account_journal_lock_date: no need to inherit write()

Commit:
[UPD] Update account_journal_lock_date.pot

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/

Gelo-fl committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (20 of 20 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/es/

adrianojprado committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 20.0% (4 of 20 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/pt_BR/

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 20.0% (4 of 20 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/it/

ThijsvOers committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (20 of 20 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/nl/

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (20 of 20 strings)

Translation: account-financial-tools-16.0/account-financial-tools-16.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-16-0/account-financial-tools-16-0-account_journal_lock_date/sv/

luisDIXMIT committed:
[IMP] account_journal_lock_date: pre-commit auto fixes

luisDIXMIT committed:
[MIG] account_journal_lock_date: Migration to 17.0

Commit:
[UPD] Update account_journal_lock_date.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_journal_lock_date/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_journal_lock_date/it/

santiagopim committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (19 of 19 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_journal_lock_date/ca/

santiagopim committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (19 of 19 strings)

Translation: account-financial-tools-17.0/account-financial-tools-17.0-account_journal_lock_date
Translate-URL: https://translation.odoo-community.org/projects/account-financial-tools-17-0/account-financial-tools-17-0-account_journal_lock_date/ca/

StefanRijnhart committed:
[MIG] account_journal_lock_date: Migration to 18.0

StefanRijnhart commented:
/ocabot migration account_journal_lock_date

Saran440 reviewed (approved):
Functional Test :+1:

kanisorn-kate reviewed (commented):
Look good

Saran440 commented:
@StefanRijnhart Can you check file conflict please?

StefanRijnhart committed:
[REF] account_journal_restrict_mode: run tests separately

Fixes
```
ERROR odoo odoo.addons.account_journal_lock_date.tests.test_journal_lock_date: FAIL: TestJournalLockDate.test_journal_lock_date
odoo.exceptions.UserError: This document is protected by a hash. Therefore, you cannot edit the following fields: Number.
```

StefanRijnhart commented:
@Saran440 Thanks for noting. Rebased and conflict resolved.

StefanRijnhart commented:
@OCA/accounting-maintainers this one should be ready for merge

kittiu commented:
/ocabot merge nobump
